csaf_files/IT/white/2025/va-25-136-01.json

```
@@ -1,192 +1,205 @@
+        "title": "Legal Notice",
+        "category": "legal_disclaimer"
+      },
+      {
+        "text": "Worldwide",
+        "title": "Countries and Areas Deployed",
+        "category": "other"
+      },
+      {
+        "text": "Information Technology",
+        "title": "Critical Infrastructure Sectors",
+        "category": "other"
+      },
+      {
+        "text": "IBM Security Guardium 11.5 is vulnerable to stored cross-site scripting (XSS). An attacker with administrative privileges could store arbitrary content or script in the administrative web interface that would be rendered or executed by users visiting modified parts of the web interface. The attacker could modify the web interface or obtain session tokens which could be used to log in as a different user.\n\n, which could allow a privileged user to disclose otherwise protected credentials. ",
+        "title": "Risk Evaluation",
+        "category": "summary"
+      },
+      {
+        "text": "Apply Guardium Data Protection patch 11.0p565 for Guardium 11.5.",
+        "title": "Recommended Practices",
+        "category": "general"
+      },
+      {
+        "text": "United States",
+        "title": "Company Headquarters Location",
+        "category": "other"
+      }
+    ],
+    "references": [
+      {
+        "url": "https://raw.githubusercontent.com/cisagov/CSAF/develop/csaf_files/IT/white/2025/va-25-136-01.json",
+        "summary": "Vulnerability Advisory VA-25-136-01 CSAF",
+        "category": "self"
+      }
+    ]
+  },
+  "product_tree": {
+    "branches": [
+      {
+        "category": "vendor",
+        "name": "IBM",
+        "branches": [
+          {
+            "category": "product_name",
+            "name": "Security Guardium",
+            "branches": [
+              {
+                "category": "product_version_range",
+                "name": "<11.5 11.0p565",
+                "product": {
+                  "name": "IBM Security Guardium <11.5 11.0p565",
+                  "product_id": "CSAFPID-0001"
                 }
-            },
-            "id": "VA-25-136-01",
-            "initial_release_date": "2025-05-16T20:14:51Z",
-            "status": "final",
-            "version": "1.0.0",
-            "revision_history": [
-                {
-                    "number": "1.0.0",
-                    "summary": "Initial publication",
-                    "date": "2025-05-16T20:14:51Z"
+              },
+              {
+                "category": "product_version",
+                "name": "11.5 11.0p565",
```

Patch identifier should not be included in the version.
```suggestion
                "name": "11.5",
```